### ISSUE TITLE
User Deprecated in EightPoints\Bundle\GuzzleBundle\DataCollector\HttpDataCollector method collect()

### ISSUE THREAD

**Issue by dmamchyts** [8p/EightPointsGuzzleBundle#330]
| Q                | A
| ---------------- | -----
| Symfony version  | ^6.3
| Bundle version   | ^8.4

Hello.
Deprecation error: 
```
User Deprecated: Method "Symfony\Component\HttpKernel\DataCollector\DataCollectorInterface::collect()" might add "void" as a native return type declaration in the future. Do the same in implementation "EightPoints\Bundle\GuzzleBundle\DataCollector\HttpDataCollector" now to avoid errors or add an explicit @return annotation to suppress this message.
```

![2023-09-11_11-12](https://github.com/8p/EightPointsGuzzleBundle/assets/102530135/fbf0b4ec-7c44-4251-938f-68f79f7169c3)

**Comment by ajaypatidar-dataaxle** [comment]
Getting similar error

`1x: Method "Symfony\Component\HttpKernel\Bundle\Bundle::boot()" might add "void" as a native return type declaration in the future. Do the same in child class "EightPoints\Bundle\GuzzleBundle\EightPointsGuzzleBundle" now to avoid errors or add an explicit @return annotation to suppress this message.`

Please help

**Comment by ajaypatidar-dataaxle** [comment]
> Fixed here #339

OK, Thank you. Please update the same on packagist.org
